app/assets/scripts/components/highlighted-operations/index.js

```diff
@@ -75,12 +79,12 @@ class HighlightedOperations extends React.Component {
     if (fetched && (error || !Array.isArray(data.results) || !data.results.length)) return null;
     else if (!fetched || fetching) return <div className='inner'><Fold title={title}><BlockLoading/></Fold></div>;
     let operations = data.results;
-    if (this.props.opsType === 'region') {
-      operations = operations.filter(op => countriesByRegion[this.props.opsId].includes(op.countries[0].id.toString()));
-    }
-    if (this.props.opsType === 'country') {
-      operations = operations.filter(op => op.countries[0].id === this.props.opsId);
-    }
+    // if (this.props.opsType === 'region') {
+    //   operations = operations.filter(op => countriesByRegion[this.props.opsId].includes(op.countries[0].id.toString()));
+    // }
+    // if (this.props.opsType === 'country') {
+    //   operations = operations.filter(op => op.countries[0].id === this.props.opsId);
+    // }
```

Comment:
If the initial request in mount falls back to `_getFeaturedEmergencies` will we still need to access this logic?

Reply:
It should never "fallback" - if the type is `region` it would query for emergencies filtered by region, and if that request fails for some reason, it will set the error state to true for emergencies which should then be handled the same way as errors for `_getFeaturedEmergencies`. Does that make sense?

Reply:
I left these lines commented out in case we want to use this logic to still show "Highlighted Operations" on the country page, but those would have to be operations highlighted for the region - I don't think we want to add another "Highlighted for Country" flag in the backend. But thinking about this, I would really vote strongly to remove from the country page.

Reply:
ah yes, sorry I missed the fact that we are are using these actions differently